pkg/doc/jsonld/contexts.go

```diff
@@ -122,4 +124,9 @@ var embedContexts = []ContextDocument{ //nolint:gochecknoglobals
 		DocumentURL: "https://ns.did.ai/suites/secp256k1-2019/v1/",
 		Content:     secp256k12019,
 	},
+	{
+		URL:         "https://w3id.org/citizenship/v1",
```

Comment:
This context cannot be embedded into the core framework loader - needs to be loaded by an application or test scenario.